code/modules/admin/verbs/playsound.dm

```diff
@@ -73,15 +73,20 @@
 		if(length(web_sound_input))
 
 			web_sound_input = trim(web_sound_input)
+			var/static/regex/html_protocol_regex = regex("https?://")
+			if(findtext(web_sound_input, ":") && !findtext(web_sound_input, html_protocol_regex))
+				to_chat(src, "<span class='boldwarning'>Non-http(s) URIs are not allowed.</span>")
+				to_chat(src, "<span class='warning'>For youtube-dl shortcuts like ytsearch: please use the appropriate full url from the website.</span>")

```

Comment:
It occurs to me you could manually allow these, since youtube would obviously parse them

Reply:
There are a lot more websites than youtube supported by youtube-dl, I don't want to whitelist every one.